appservice/src/intTest/groovy/nu/yona/server/ActivityTest.groovy

```diff
@@ -26,17 +29,23 @@ class ActivityTest extends AbstractAppServiceIntegrationTest
 		response.responseData._embedded."yona:dayActivityOverviews".size() == 1
 		def dayActivityOverview = response.responseData._embedded."yona:dayActivityOverviews"[0]
 		dayActivityOverview._embedded."yona:dayActivities"
-		dayActivityOverview._embedded."yona:dayActivities".size() == 1
-		def dayActivityForGoal = dayActivityOverview._embedded."yona:dayActivities"[0]
-		dayActivityForGoal.spread
-		dayActivityForGoal.spread.size() <= 96
-		dayActivityForGoal.totalActivityDurationMinutes == 1
-		dayActivityForGoal.goalAccomplished == false
-		dayActivityForGoal.totalMinutesBeyondGoal == 1
-		dayActivityForGoal.date =~ /\d{4}\-\d{2}\-\d{2}/
-		dayActivityForGoal.timeZoneId == "Europe/Amsterdam"
-		dayActivityForGoal._links."yona:goal"
-		dayActivityForGoal._links.self
+		dayActivityOverview._embedded."yona:dayActivities".size() == 2
+		def dayActivityForBudgetGoal = dayActivityOverview._embedded."yona:dayActivities".find{ it._links."yona:goal".href != timeZoneGoal.url}
+		dayActivityForBudgetGoal
+		!dayActivityForBudgetGoal.spread
+		dayActivityForBudgetGoal.totalActivityDurationMinutes == 1
+		dayActivityForBudgetGoal.goalAccomplished == false
+		dayActivityForBudgetGoal.totalMinutesBeyondGoal == 1
+		dayActivityForBudgetGoal.date =~ /\d{4}\-\d{2}\-\d{2}/
+		dayActivityForBudgetGoal.timeZoneId == "Europe/Amsterdam"
+		dayActivityForBudgetGoal._links."yona:goal"
+		dayActivityForBudgetGoal._links.self
+
+		//time zone goal should have spread
+		def dayActivityForTimeZoneGoal = dayActivityOverview._embedded."yona:dayActivities".find{ it._links."yona:goal".href == timeZoneGoal.url}
+		dayActivityForTimeZoneGoal
+		dayActivityForTimeZoneGoal.spread
+		dayActivityForTimeZoneGoal.spread.size() <= 96
```

Comment:
Shouldn't this be exactly 96? We are reporting all 15-minute blocks of the day, right?